<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lanzador</class>
 <widget class="QWidget" name="Lanzador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dune Remake</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-1</y>
         <width>341</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CONECTAR AL SERVIDOR</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>181</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>DIRECCIÃ“N DEL SERVIDOR</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtDireccionServidor">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>210</width>
         <height>29</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>110</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>PUERTO</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtPuertoServidor">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>130</y>
         <width>99</width>
         <height>29</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnConectar">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>170</y>
         <width>181</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>CONECTAR</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listServidoresRecientes">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>321</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>310</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>SERVIDORES RECIENTES</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>341</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ELEGIR SALA</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QListWidget" name="listSalasDisponibles">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>331</width>
         <height>151</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>331</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>SALAS DISPONIBLES</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>370</y>
         <width>331</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>CREAR UNA NUEVA SALA</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtNombreSala">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>390</y>
         <width>331</width>
         <height>29</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Nombre de la sala</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCrearSala">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>470</y>
         <width>201</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>CREAR SALA</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnElegirSala">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>290</y>
         <width>111</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>UNIRSE</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnActualizar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>290</y>
         <width>111</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>ACTUALIZAR</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cbMapas">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>430</y>
         <width>271</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>435</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>MAPA:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>CONECTADO A:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblNombreSala">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>330</y>
         <width>211</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>0</y>
         <width>341</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>INICIAR JUEGO</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>310</y>
         <width>331</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>JUGADORES EN LA SALA</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listJugadoresEnSala">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>330</y>
         <width>321</width>
         <height>171</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="btnNombreJugador">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>331</width>
         <height>29</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>321</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>MI NOMBRE</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>331</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="rbAtreides">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>ATREIDES</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbHarkonnen">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>HARKONNEN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbOrdos">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>ORDOS</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>1024</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnIniciarJuego">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>0</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INICIAR JUEGO</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbPantallaCompleta">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator:checked {
	image: url(:/imgs/nuevos/check-square-solid.svg);
	height: 32px;
    width: 32px;
}

QCheckBox::indicator:unchecked {
	image: url(:/imgs/nuevos/square-solid.svg);
	height: 32px;
    width: 32px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PANTALLA COMPLETA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>VSYNC</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbVsync">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator:checked {
	image: url(:/imgs/nuevos/check-square-solid.svg);
	height: 32px;
    width: 32px;
}

QCheckBox::indicator:unchecked {
	image: url(:/imgs/nuevos/square-solid.svg);
	height: 32px;
    width: 32px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbSFX">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator:checked {
	image: url(:/imgs/nuevos/check-square-solid.svg);
	height: 32px;
    width: 32px;
}

QCheckBox::indicator:unchecked {
	image: url(:/imgs/nuevos/square-solid.svg);
	height: 32px;
    width: 32px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SONIDO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbMusica">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator:checked {
	image: url(:/imgs/nuevos/check-square-solid.svg);
	height: 32px;
    width: 32px;
}

QCheckBox::indicator:unchecked {
	image: url(:/imgs/nuevos/square-solid.svg);
	height: 32px;
    width: 32px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>MUSICA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../data/rc-cliente/recursos.qrc"/>
 </resources>
 <connections/>
</ui>
